tests/browser/spa/hashchange-popstate.browser.js

```diff
@@ -4,7 +4,7 @@
  */
 
 const jil = require('jil')
-let cleanUrl = require('../../../agent/clean-url')
+let cleanUrl = require('@newrelic/browser-agent-core/src/common/url/clean-url').cleanURL
```

Comment:
Just double checking that this doesn't get its own tests/lib version, since `cleanUrl` is used for both "actual" and "expected" output comparison.